zap/src/main/java/org/zaproxy/zap/extension/brk/BreakpointsOptionsPanel.java

```diff
@@ -129,7 +129,7 @@ public void initParam(Object obj) {
 
         getCheckBoxConfirmDropMessage().setSelected(param.isConfirmDropMessage());
         // Note param.alwaysOnTop will be null if the user hasn't specified a preference yet
-        getCheckBoxAlwaysOnTop().setSelected(param.getAlwaysOnTop() != Boolean.FALSE);
+        getCheckBoxAlwaysOnTop().setSelected(!param.getAlwaysOnTop().equals(Boolean.FALSE));
```

Comment:
This can lead to NPE, better to swap the check.

Reply:
You mean `!Boolean.FALSE.equals(param.getAlwaysOnTop())`?

Reply:
Yes.